docs/api-docs/getting-started/api-status-codes.md

```diff
@@ -39,15 +39,17 @@ The BigCommerce API responds to requests with different HTTP status codes depend
 
 | Code | Text | Purpose |
 |-|-|-|
-| **400** | **Bad Request** | Issued when a malformed request was sent.
+| **400** | **Bad Request** | Issued when a malformed request was sent. |
+||| The request can not be completed due to a URL restriction. Check the URL for ports that may conflict with site permissions. |

```

Comment:
When I preview this doc, there's something off about the table formatting. Can you take a look again?

Reply:
Definitely, I'll take a look at this.

Reply:
Fixed? Maybe? Preview looks better.